iohk/iohk-ops.hs

```diff
@@ -313,9 +311,6 @@ runNew o@Options{..} New{..} args = do
   echo $ "-- " <> (unsafeTextToLine $ configFilename) <> " is:"
   cmd o "cat" [configFilename]
 
```

Comment:
There should be a call to create here. The idea is to make sure it's either created or modified. We also want to bring create back to backwards compatibitily

Reply:
@domenkozar, `io deploy` runs `io modify` (which should be probably renamed into `update`), which runs either `nixops modify` or `nixops create`, as appropriate.

Also, `io create` is still there and it runs `io modify`.